src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1652,6 +1652,20 @@ internal void SetupWin32Resources(CommonPEModuleBuilder moduleBeingBuilt, Stream
 
         #region Emit
 
+        /// <summary>
+        /// There are two ways to sign PE files
+        ///   1. By directly signing the <see cref="PEBuilder"/>
+        ///   2. Write the unsigned PE to disk and use CLR COM APIs to sign.
+        /// The preferred method is #1 as it's more efficient and more resilient (no reliance on %TEMP%). But 
+        /// we must continue to support #2 as it's the only way to do the following:
+        ///   - Access private keys stored in a key 
```

Comment:
>key [](start = 48, length = 3)

Is this really the end of the sentence?

Reply:
Yes. It's just a terrible sentence. Fixed to say "container".

---
In reply to: [245379071](https://github.com/dotnet/roslyn/pull/31986#discussion_r245379071) [](ancestors = 245379071)